tests/Test/Updater/UpdaterTest.php

```diff
@@ -11,7 +11,17 @@ class UpdaterTest extends TestCase
 {
     public static function tearDownAfterClass(): void
     {
-        array_map('unlink', glob(__DIR__ . '/fixtures/generated/*.*'));
+        $files = new \RecursiveIteratorIterator(
+            new \RecursiveDirectoryIterator(__DIR__ . '/fixtures/generated', \RecursiveDirectoryIterator::SKIP_DOTS),
+            \RecursiveIteratorIterator::CHILD_FIRST
+        );
+        foreach ($files as $file) {
+            if ($file->isDir()) {
+                rmdir($file);
+            } else {
+                unlink($file);
+            }
+        }
```

Comment:
FYI I introduced a `Workspace` class which does this sort of thing for tests (and isn't confusing in this project):

```
// abstract test class in test directory
public function workspace(): Workspace
{
   $workspace = new Workspace(__DIR__ . '/Workspace');
}
```

```
// setup
$this->workspace()->reset();

// test
$this->workspace()->...
````